app/script/view_model/content/InputBarViewModel.js

```diff
@@ -398,15 +400,40 @@ z.viewModel.content.InputBarViewModel = class InputBarViewModel {
         textarea.selectionEnd = newEnd;
       }
 
+      this.selectionStart(newStart);
+      this.selectionEnd(newEnd);
+
       const startFlowRegexp = /\B@$/;
       if (startFlowRegexp.test(text)) {
         this.isInMentionFlow(true);
       }
     }
   }
 
+  updateMentions(data, event) {
+    const textarea = event.target;
+    const value = textarea.value;
+    const previousValue = this.input();
+    const mentions = this.parseForMentions(previousValue, this.conversationEntity().participating_user_ets());
+    const lengthDifference = value.length - previousValue.length;
+    this.logger.log(this.updateMentionRanges(mentions, this.selectionStart(), this.selectionEnd(), lengthDifference));
```

Comment:
Can we add some helpful text to the log? Like "Mention ranges: ..."?

Reply:
This is debug code that will be removed in the very near future ;)

Reply:
This line is just there to consume it for now. It will be gone in the next PR.